DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/projectingIndividualFields/metadata.json

```diff
@@ -0,0 +1,126 @@
+    "SourceFileName": "ProjectingIndividualFields.java",
+    "Title": "Projecting Individual Fields",
+    "DescriptionLines": [
+        "**Projections** are used in a query when you need to **retrieve only specific document fields** from the server, instead of the whole, full document.",
+        "(When projections are Not defined - the full document content is returned).<br>",
+        "This reduces the amount of data sent over the network and is useful when only partial data is needed.",
+        "",       
+        "The returned entities field names can be customized to suit your needs.",
+        "",
+        "The output projected entities returned by the query are Not tracked by the Session, as they are Not full documents.",
+        "",
+        "Projections are applied at the last stage in the query pipeline - after documents have been processed, filtered, sorted and paged."
+    ],
+
+    "Assets": [
+        {
+            "Title": "Querying : Projections",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/querying/projections"
+        },
+        {
+            "Title": "How to project query results",
+            "Type" : "link",
+            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/querying/how-to-project-query-results"
+        },
+        {
+            "Title": "Projecting data in queries",
+            "Type" : "link",
+            "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch04/Ch04.md#projecting-data-in-queries"
+        },
+        {
+            "Title": "Projecting results",
+            "Type" : "link",
+            "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch09/Ch09.md#projecting-results"
+        },
+        {
+            "Title": "Sample Program",
+            "Type" : "link",
+            "Url"  : "https://github.com/ravendb/ravendb-jvm-tutorials/blob/hospital/README.md#4-aggregating-and-projecting"
+        }
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Define a query",
+            "Slug": "step-1",
+            "DescriptionLines": [
+                "* Define a query on the 'Companies' collection Using the Session `query()` method."
+            ],
+            "Assets": [
+                {
+                    "Title": "Query Overview Demo",
+                    "Type" : "demo",
+                    "Url"  : "/java/queries/query-overview"
+                },
+                {
+                    "Title": "Sample Program",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/ravendb-jvm-tutorials/blob/hospital/README.md#4-aggregating-and-projecting"
+                }
+            ]
+        },
+        {
+            "Title": "Define a projection",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* Use `selectFields()` to project partial document data from the server.",
```

Comment:
I wonder if we should mention about second overload - when we specify only target object class.

Reply:
![image](https://user-images.githubusercontent.com/17376974/72424082-15de5f00-378e-11ea-8b2a-45a763729922.png)

Reply:
I wonder if sentense (When custom class fields are subset...) is necessary.

Like we talked it takes list and fills query data.

Reply:
removed that  line